textRevers/src/main/java/app/textRevers/CharsCounter.java

```diff
@@ -0,0 +1,52 @@
+package app.textRevers;
+
+
+import java.util.Map;
+import java.util.LinkedHashMap;
+
+/**
+ * @author Anastasiia Nudha
+ * @version 0.1.0
+ * @since 0.0.1
+ */
+
+public abstract class CharsCounter {
+
+    /**
+     * Method name: countChars
+     *
+     * @param sentence The sentence in which be counted chars.
+     * @return (Map<Character, Integer>) Map with key of character and count of this character.
+     * 											Inside the function: 1. Count characters; 2. Return
+     * 											 counted character Map;
+     */
+
+    public static Map<Character,Integer> countChars(String sentence){
+
+        Map<Character, Integer> map = new LinkedHashMap<>();
+
+
+        if(sentence==null){
+            throw new IllegalArgumentException("Text can't be null, please enter a text");
+        }
+
+        if(!CreateOutputLook.getCache().containsKey(sentence)){
+
```

Comment:
Too many enters. Read Code Conventions that i send you please https://www.oracle.com/technetwork/java/codeconventions-150003.pdf